src/ring/middleware/anti_forgery.clj

```diff
@@ -35,21 +17,15 @@
       (-> request :headers (get "x-csrf-token"))
       (-> request :headers (get "x-xsrf-token"))))
 
-(defn- valid-token? [request read-token]
-  (let [user-token   (read-token request)
-        stored-token (session-token request)]
-    (and user-token
-         stored-token
-         (crypto/eq? user-token stored-token))))
-
 (defn- get-request? [{method :request-method}]
   (or (= method :head)
       (= method :get)
       (= method :options)))
 
-(defn- valid-request? [request read-token]
+(defn- valid-request? [state-management-strategy request read-token]
```

Comment:
Change `state-management-strategy` to `strategy`. There's only one type of strategy, so we don'tneed to be overly verbose.